src/backend/marsha/e2e/test_lti.py

```diff
@@ -348,12 +349,34 @@ def test_lti_video_play(page: Page, live_server: LiveServer, mock_video_cloud_st
     ) as response_info:
         assert 200 == response_info.value.response().status
 
+    # Wait player initialization before playing it
+    page.expect_request(
+        lambda request: request.method == "POST"
+        and re.match(r"^.*/xapi/video/$", request.url)
+        and request.post_data_json.get("verb").get("display").get("en-US")
+        == "initialized"
+    )
```

Comment:
Maybe we could make it less verbose with something like
```suggestion
    expect_xapi_event(page, "initialized")
```
And by adding a function outside of the testcase:
```python
def expect_xapi_event(page, verb):
    page.expect_request(
        lambda request: request.method == "POST"
        and re.match(r"^.*/xapi/video/$", request.url)
        and request.post_data_json.get("verb").get("display").get("en-US")
        == verb
    )
```